### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/content_conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#229]
```
hei Oskar, I have added ALL italian translation of conversationlist!

to when a patch for the market??

good job Oskar
```

Original issue reported on code.google.com by `k6b...@gmail.com` on 3 Aug 2011 at 10:12

Attachments:
- [content_conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-229/comment-0/content_conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Committed in r194. Nice work!
```

Original comment by `oskar.wi...@gmail.com` on 30 Oct 2011 at 1:13
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.11**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
